iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/metrics/FileMetrics.java

```diff
@@ -333,149 +331,236 @@ private long getOpenFileHandlersNumber() {
   }
 
   // following are update functions for tsfile metrics
-  public void addFile(long size, boolean seq, String name) {
-    updateGlobalCountAndSize(size, 1, seq);
+  public void addFile(String database, String regionId, long size, boolean seq, String name) {
+    updateGlobalCountAndSize(database, regionId, size, 1, seq);
     try {
       TsFileNameGenerator.TsFileName tsFileName = TsFileNameGenerator.getTsFileName(name);
       int level = tsFileName.getInnerCompactionCnt();
-      updateLevelCountAndSize(size, 1, seq, level);
+      updateLevelCountAndSize(size, 1, seq, level, database, regionId);
```

Comment:
Put the database and regionID arguments first, just like all the other functions such as `updateGlobalCountAndSize`

Reply:
Get it~